pages/_app.tsx

```diff
@@ -1,17 +1,39 @@
-import "../styles/globals.css";
 import "@rainbow-me/rainbowkit/styles.css";
+import { ChakraProvider } from "@chakra-ui/provider";
+import { Grid, GridItem } from "@chakra-ui/layout";
+import { extendTheme } from "@chakra-ui/theme-utils";
 import type { AppProps } from "next/app";
+import Head from "next/head";
 import { RainbowKitProvider, getDefaultWallets } from "@rainbow-me/rainbowkit";
 import { configureChains, createClient, WagmiConfig } from "wagmi";
-import { mainnet, polygon, optimism, arbitrum, goerli } from "wagmi/chains";
+import {
+  mainnet,
+  polygon,
+  optimism,
```

Comment:
Can remove this unused import

Reply:
Yes, and we should add a prettier check and command to catch this stuff prior to review

Reply:
Good call, can you create an issue to setup CI that runs prettier/linters/builds the app? We can add tests too of course once we have that. Up to you whether to adds this on as part of https://github.com/ScopeLift/railgun-shielder/issues/2 and do that early, or to handle it separately from the deployments

Reply:
Created an issue #7